configs/components/marionette-collective.rb

```diff
@@ -66,7 +66,28 @@
   when "aix"
     pkg.install_service "resources/aix/mcollective.service", nil, "mcollective"
   when "windows"
-    puts "Service files not enabled on windows"
+    service_hash = [{
+      :directory_ref => "bindir",
```

Comment:
This will be the bindir directory id that's used to define the bindir in the directory wix file